src/main/java/com/icthh/xm/tmf/ms/qualification/config/WebMvcConfiguration.java

```diff
@@ -12,21 +12,20 @@
 @Configuration
 public class WebMvcConfiguration extends XmWebMvcConfigurerAdapter {
 
-    private final LepInterceptor lepInterceptor;
     private final ApplicationProperties appProps;
+    private final TenantVerifyInterceptor tenantVerifyInterceptor;
 
-    protected WebMvcConfiguration(LepInterceptor lepInterceptor,
-        TenantInterceptor tenantInterceptor,
-        XmLoggingInterceptor xmLoggingInterceptor,
-        ApplicationProperties appProps) {
+    protected WebMvcConfiguration(TenantInterceptor tenantInterceptor,
+                                  XmLoggingInterceptor xmLoggingInterceptor,
+                                  ApplicationProperties appProps, TenantVerifyInterceptor tenantVerifyInterceptor) {
         super(tenantInterceptor, xmLoggingInterceptor);
-        this.lepInterceptor = lepInterceptor;
         this.appProps = appProps;
+        this.tenantVerifyInterceptor = tenantVerifyInterceptor;
     }
 
     @Override
     protected void xmAddInterceptors(final InterceptorRegistry registry) {
-        registry.addInterceptor(lepInterceptor).addPathPatterns("/**");
+        registerTenantInterceptorWithIgnorePathPattern(registry, tenantVerifyInterceptor);
```

Comment:
try to not remove lepInterceptor

register lepInterceptor using registerTenantInterceptorWithIgnorePathPattern but AFTER tenantVerifyInterceptor